plugins/cdi/src/main/java/org/apache/struts2/cdi/CdiObjectFactory.java

```diff
@@ -173,7 +174,9 @@ protected InjectionTarget<?> getInjectionTarget(Class<?> clazz) {
         InjectionTarget<?> result;
         result = injectionTargetCache.get(clazz);
         if (result == null) {
-            result = beanManager.createInjectionTarget(beanManager.createAnnotatedType(clazz));
+			final InjectionTargetFactory<?> injectionTargetFactory =
```

Comment:
I noticed this file has mixed use of 4-space indents and tab indents. Let's consolidate on 4-space indents as that is what is used in the majority of this repo.

Reply:
Done :)